### DISCUSSION TITLE
Add another user?

### DISCUSSION THREAD

**Discussion by revilo951** [vrtmrz/obsidian-livesync#530] [Q&A]
I'd like to add another user to my DB - is there a way to do this? Do I need a whole other db?

To clarify, I don't want to share my notes with them, I want them to be able to sync their own notes.

**Comment by mgmorice** [comment]
Did you ever find an answer to this?  
I switched to Obsidian this month and am smitten...so much that my college age son wants his own setup with it.  
If I have to self host a second instance of this, so be it, but if there's an easier method I'm all for it.
